pkgs/development/libraries/SDL2/default.nix

```diff
@@ -92,7 +95,8 @@ stdenv.mkDerivation rec {
     ++ optional pulseaudioSupport libpulseaudio
     ++ optional udevSupport udev
     ++ optionals waylandSupport [ wayland wayland-protocols libxkbcommon ]
-    ++ optionals x11Support [ libICE libXi libXScrnSaver libXcursor libXinerama libXext libXrandr libXxf86vm ];
+    ++ optionals x11Support [ libICE libXi libXScrnSaver libXcursor libXinerama libXext libXrandr libXxf86vm ]
+    ++ optionals drmSupport [ libdrm mesa ];
 
```

Comment:
Doesn't this make the closure a lot bigger? And does it matter for people not using mesa?

Reply:
Not really - SDL2 is currently 519.8M on unstable-small, and 520.3M with this change.

Reply:
Also re: people not using Mesa, I _think_ this might work with Mesa 22 and latest Nvidia stuff, but I don't have any hardware to test on.

Reply:
It does not.  This derivation is very small.

Reply:
What I tried (`nix-store -qR ./result | xargs du -sc | sort -n`):
```diff
-560744  total
+561260  total
```

Reply:
Is this on staging?

Reply:
I checked that on master with this commit cherry-picked.

Reply:
Oh, I didn't see your replies before sending mine.  A stuck tab, apparently.

Reply:
I wonder why you're getting 40M more than me then. I'm on x86_64-linux.

Reply:
560744/1024 = 547.6 and `du` added some rounding (allocation overhead).
```
$ nix path-info -S ./result
/nix/store/pyksj4gj6bjdsgfq3slfl28pa00h92vl-SDL2-2.0.20   545081840  = 519.8 MiB
```

Reply:
OK, that seems to line up with what I'm seeing. Either way, the difference is under a megabyte, so probably nothing to worry about.